src/components/Menus.ts

```diff
@@ -187,7 +187,7 @@ export class Menus<TGameStartr extends FullScreenPokemon> extends Component<TGam
 
             this.gameStarter.menuGrapher.createMenu("Pause", {
                 size: {
-                    height: 64
+                    height: 255
```

Comment:
`256` is correct, actually. The issue is that it's off by a factor of 4.

There used to be a concept of "unitsize" where every size measurement was multiplied by `FSP.unitsize`, which was 4. It was supposed to make it easier to zoom in-game... but ended up complicating a whole lot of stuff. I must have missed this line of code in the conversion.